core/src/main/java/io/temporal/samples/nexus/README.MD

```diff
@@ -33,7 +33,7 @@ This sample shows how to use Temporal for authoring a Nexus service and call it
 > HTTP port is required for Nexus communications
 
 ```
-temporal server start-dev --http-port 7243 --dynamic-config-value system.enableNexus=true
+temporal server start-dev --http-port 7243
```

Comment:
Don't need the http port anymore for workflow-to-workflow calls. The CLI will allocate a random port.

Reply:
Will change